python/grass/pygrass/vector/__init__.py

```diff
@@ -108,7 +108,7 @@ def __next__(self):
 
     @must_be_open
     def next(self):
-        return self.__next__()
+        return next(self)
```

Comment:
Same as comment above...

Reply:
Changes from https://github.com/echoix/grass/commit/72dfc4c48aed0898f1a045a97831f4b14863ea51 were for adding python3 support, so it goes in the same idea as below

Reply:
How do we make sure CI runs doctests? as the __next__() has a doctest that uses `.next()` and if it should complain about missing that function it should be there
```python
            >>> test_vect = Vector(test_vector_name)
            >>> test_vect.open(mode="r")
            >>> test_vect.next()
            Point(10.000000, 6.000000)
            >>> test_vect.next()
            Point(12.000000, 6.000000)
            >>> test_vect.close()
```

Reply:
Looks like they were necessary (multiple tests fails, the doctests) and didn't cause an circular loop finally, as my first change

Reply:
So `.next()` was part of a public API, so, I needed to keep it there, as even doctests were using them.